ast2vec/repo2/base.py

```diff
@@ -130,13 +137,13 @@ def thread_loop(thread_index):
                                 queue_out.put_nowait(None)
                                 continue
 
-                            uast = self._bblfsh[thread_index].parse(
+                            response = self._bblfsh[thread_index].parse(
                                 filepath, language=language, timeout=self._timeout)
-                            if uast is None:
+                            if response is None:
                                 self._log.warning("bblfsh timed out on %s", filepath)
                             queue_out.put_nowait(Namespace(filepath=filepath,
```

Comment:
One last thing: this is not what Namespace is intented for. Please use `collections.NamedTuple` and declare the type in base.py

Sorry if I missed this before.